jans-auth-server/common/pom.xml

```diff
@@ -219,5 +219,15 @@
 			<groupId>org.testng</groupId>
 			<artifactId>testng</artifactId>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-testng</artifactId>
+            <version>0.4.8</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
Worths checking if we can include it as part of a lower level, perhaps bom or similar level.

Reply:
I do agree, version should go to at least parent pom.

Reply:
I have removed the version from jans-common pom,xml, and put it on parent pom.xml.
I have included the dependency in jans-bom pom.xml too